indra/assemblers/pybel/assembler.py

```diff
@@ -1,19 +1,21 @@
 from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
-import uuid
+
 import logging
+import uuid
+from copy import copy, deepcopy
+
 import networkx as nx
-from copy import deepcopy, copy
+from builtins import dict, str
```

Comment:
Why move this stuff around?

Reply:
I used the reformat code function in pycharm and one of the things it takes care of is to sort the imports. I accidentally had PyBEL imported before so it didn't get the order quite right (it assumes if you have other projects open then they should be in the last group of project-related imports)

I've fixed that